shaders/src/common_graphics.fs

```diff
@@ -111,4 +111,7 @@ vec3 uintToColorDebug(uint v) {
     } else if (v == 5u) {
         return vec3(0.0, 1.0, 1.0);     // cyan
     }
+
+    // fallback to handle "not all code-paths return" warnings
+    return vec3(0.0, 0.0, 0.0);
 }
```

Comment:
I still get a warning like this at startup:
```
program_source:431:1: warning: control may reach end of non-void function
```
Also, I'd prefer if the unused variable warning got fixed in this ticket too, it's such a small thing compared to the overhead of administrating and bumping twice.